Generate properties based on `getPropTable`

pablohpsilva opened the pull request:
In this PR I've created a small script that will create the `interface CCSSProps` for us based on the `getPropTable` array.

pablohpsilva commented on packages/core/src/createProps.ts:
```diff
@@ -107,7 +107,7 @@ export const getPropTable = (): [string, string, string, CCSSParser?, CCSSParser
 
         // Grid
-        ['g', 'gr', 'grid'],
+        ['g', 'grd', 'grid'],
```
I renamed it because we had two entries with the same `gr` name. This was causing issues after creating the interface.

wintercounter replied:
Good, I was afraid we will have such :)

wintercounter commented:
It's awesome. Just one thing. Please add short, light and long to props so we'll be able to display all properties. You can rename prop to long, we will use long to fetch the data.

pablohpsilva commented:
I took the liberty to change the `propDoc` and add the following changes:
- Since we're using multiple props (short + aliases) now, I avoid unnecessary requests by memoizing a fetch function;
- I fixed the comments spacings and tabbing. It was REALLY triggering me;
- Added a new script on `package.json` to generate the properties;
- Did the requested changes for the `propDoc(Start|End)` as requested;
- Cleaned up the `prodDoc` file.
